tests/network-ovn

```diff
@@ -1091,6 +1097,10 @@ EOF
     dig a +tcp @192.0.2.1 u2.lxd
     dig aaaa +tcp @2001:db8:1:2::1 u2.lxd
 
+    echo "==> Check accessing load balancer's listen address inside the u2"
+    lxc exec u2 -- dig a +tcp @192.0.2.1 lxd.localdomain +short
```

Comment:
[ "$(lxc exec u1 -- dig a +tcp @192.0.2.1 lxd.localdomain +short)" = "127.0.0.2" ]

Reply:
Oh my bad, I assumed IPv6 was used only for the DNS server address but you are right, it's making a `AAAA` request. Adding `AAAA` synthetic records would work. Thanks :)

Reply:
The AAAA request isn't strictly needed here, we could have done an A request over ipv6 to test the ipv6 load balancer was working. But either approach is fine.